### PR TITLE
Issue #544 Recognize if bundle is embedded or not in integration tests

### PR DESCRIPTION
**Pull request by jsliacan**
I couldn't find any metadata of the `crc` binary that would identify an embedded bundle (except for file size). So, based on the output of `crc setup`, a variable `bundleEmbedded` is set to `true` or `false` and CRC is always started with `-b` flag if `false` and without it if `true`.

### CODE REVIEW

**Comment by anjannath** [comment]
@jsliacan There's the `constant.BundleEmbedded()` function for this check, could be useful.
https://github.com/code-ready/crc/blob/d61e39cffd1f741b7afd4d98d602431ed60dcb79/pkg/crc/constants/constants.go#L72

**Comment by jsliacan** [comment]
@anjannath thanks, it turned out to be much simpler that way.

File: test/integration/crcsuite/crcsuite.go [hunk 1]
```diff
@@ -223,15 +224,28 @@ func LoginToOcClusterSucceedsOrFails(expected string) error {
...
 func StartCRCWithDefaultBundleAndDefaultHypervisorSucceedsOrFails(expected string) error {
 
-	cmd := fmt.Sprintf("crc start -b %s -p '%s' --log-level debug", bundleName, pullSecretFile)
+	var cmd string
+
+	if constants.BundleEmbedded() {
+		cmd = fmt.Sprintf("crc start -p '%s' --log-level debug", pullSecretFile)
+	} else {
+		cmd = fmt.Sprintf("crc start -b %s -p '%s' --log-level debug", bundleName, pullSecretFile)
+	}
```
Comment (cfergeau) [inline test/integration/crcsuite/crcsuite.go]: Maybe be slightly simpler as:
```
var extraBundleArgs string
if !constants.BundleEmbedded() {
    cmd =  fmt.Sprintf(" -b %s", bundleName
}
cmd =  fmt.Sprintf("crc start -p '%s' --log-level debug %s", pullSecretFile, extraBundleArgs)

File: test/integration/features/basic.feature [hunk 2]
```diff
@@ -60,7 +60,7 @@ Feature: Basic test
     
     @darwin @linux @windows
     Scenario: CRC status check
-        When with up to "10" retries with wait period of "1m" command "crc status" output should not contain "Stopped"
+        When with up to "15" retries with wait period of "1m" command "crc status" output should not contain "Stopped"
```
Comment (cfergeau) [inline test/integration/features/basic.feature]: This change seems unrelated?

Comment (jsliacan) [reply test/integration/features/basic.feature]: Yes, completely unrelated - I just noticed that the PR was not getting through because this step fails. We discussed it on Slack today - some operators take time to come up. They are needed in the next steps, so longer wait seems reasonable. It didn't seem big enough change to warrant a separate PR - was I wrong? Or should I mention it in the commit?

Comment (cfergeau) [reply test/integration/features/basic.feature]: I'd just split it, but mentioning it in the commit log would be good enough so that people know this was not an oversight

File: test/integration/crcsuite/crcsuite.go (hunk 1 above)
Comment (jsliacan) [reply test/integration/crcsuite/crcsuite.go]: @cfergeau I changed it as you suggested - not sure why it doesn't say "outdated" above the code snippet...

File: test/integration/features/basic.feature (hunk 2 above)
Comment (jsliacan) [reply test/integration/features/basic.feature]: OK, it's in the log now - next time I'll opt for a separate PR! Thanks for reviewing :)

File: test/integration/crcsuite/crcsuite.go [hunk 3]
```diff
@@ -223,15 +224,27 @@ func LoginToOcClusterSucceedsOrFails(expected string) error {
...
 func StartCRCWithDefaultBundleAndDefaultHypervisorSucceedsOrFails(expected string) error {
 
-	cmd := fmt.Sprintf("crc start -b %s -p '%s' --log-level debug", bundleName, pullSecretFile)
+	var cmd string
+	var extraBundleArgs string
+
+	if !constants.BundleEmbedded() {
+		extraBundleArgs = fmt.Sprintf("-b %s", bundleName)
+	}
+	cmd = fmt.Sprintf("crc start -p '%s' %s --log-level debug", pullSecretFile, extraBundleArgs)
```
Comment (praveenkumar) [inline test/integration/crcsuite/crcsuite.go]: :+1:
